spec/jobs/warm_up_analytics_cache_job_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe WarmUpAnalyticsCacheJob, type: :job do
+  include ActiveJob::TestHelper
+
+  let!(:facility_groups) { create_list(:facility_group, 2) }
+  let(:to_time) { Time.new(2019, 3, 31).strftime('%Y-%m-%d') }
+  let(:from_time) { (to_time.to_time - 90.days).strftime('%Y-%m-%d') }
+
+  describe '.perform_later' do
+    let(:job) { WarmUpAnalyticsCacheJob.perform_later }
+
+    it 'queues the job' do
+      expect { job }.to change(ActiveJob::Base.queue_adapter.enqueued_jobs, :size).by(1)
```

Comment:
This helper can be used here instead: https://apidock.com/rails/v4.2.7/ActiveJob/TestHelper/assert_enqueued_jobs